src/Public.Api/SuspiciousCases/SuspiciousCasesController-Detail.cs

```diff
@@ -0,0 +1,112 @@
+        /// <response code="200">Als de opvraging van het verdacht geval gelukt is.</response>
+        /// <response code="400">Als uw verzoek foutieve data bevat.</response>
+        /// <response code="403">Als u niet beschikt over de correcte rechten om deze actie uit te voeren.</response>
+        /// <response code="406">Als het gevraagde formaat niet beschikbaar is.</response>
+        /// <response code="429">Als het aantal requests per seconde de limiet overschreven heeft.</response>
+        /// <response code="500">Als er een interne fout is opgetreden.</response>
+        [HttpGet("verdachte-gevallen/{type}", Name = nameof(DetailSuspiciousCases))]
+        [ApiOrder(ApiOrder.SuspiciousCases + 2)]
+        [ProducesResponseType(typeof(SuspiciousCasesDetailResponse), StatusCodes.Status200OK)]
+        [ProducesResponseType(typeof(ValidationProblemDetails), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status403Forbidden)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status429TooManyRequests)]
+        [ProducesResponseType(typeof(ProblemDetails), StatusCodes.Status500InternalServerError)]
+        [SwaggerResponseExample(StatusCodes.Status400BadRequest, typeof(BadRequestResponseExamplesV2))]
+        [SwaggerResponseExample(StatusCodes.Status403Forbidden, typeof(ForbiddenResponseExamplesV2))]
+        [SwaggerResponseExample(StatusCodes.Status429TooManyRequests, typeof(TooManyRequestsResponseExamplesV2))]
+        [SwaggerResponseExample(StatusCodes.Status500InternalServerError, typeof(InternalServerErrorResponseExamplesV2))]
+        [HttpCacheValidation(NoCache = true, MustRevalidate = true, ProxyRevalidate = true)]
+        [HttpCacheExpiration(CacheLocation = CacheLocation.Private, MaxAge = DefaultListCaching, NoStore = true, NoTransform = true)]
+        public async Task<IActionResult> DetailSuspiciousCases(
+            [FromRoute] int type,
+            [FromQuery] int? offset,
+            [FromQuery] int? limit,
+            [FromQuery] string nisCode,
+            [FromServices] IActionContextAccessor actionContextAccessor,
+            [FromServices] ProblemDetailsHelper problemDetailsHelper,
+            [FromServices] IOptions<SuspiciousCasesOptionsV2> responseOptions,
+            [FromServices] GetSuspiciousCasesToggle suspiciousCasesToggle,
+            CancellationToken cancellationToken = default)
+        {
+            if (!suspiciousCasesToggle.FeatureEnabled)
+            {
+                return NotFound();
+            }
+
+            var contentFormat = DetermineFormat(actionContextAccessor.ActionContext);
+
+            RestRequest BackendRequest() => CreateBackendListRequest(
+                type,
+                offset,
+                limit,
+                nisCode,
+                actionContextAccessor);
+
+            responseOptions.Value.VolgendeUrl = responseOptions.Value.VolgendeUrl.Replace("{type}", type.ToString());
```

Comment:
Shouldn't we capture volgendeUrl in a variable and change that variable? Instead of changing the responseOptions object?